resources/provider/moduledata/drift/2/drift.hcl

```diff
@@ -0,0 +1,1913 @@
+    ignore_attributes = [ "network_interface_owner_id" ]
+    iac {
+      terraform {
+        type = "aws_eip"
+        identifiers = [ "allocation_id" ]
+        attribute_map = [
+          "network_interface_id=network_interface",
+          "private_ip_address=private_ip"
+        ]
+      }
+    }
+  }
+
+  resource "ec2.flow_logs" {
+    iac {
+      terraform {
+        type = "aws_flow_log"
+      }
+    }
+  }
+
+  resource "ec2.images" {
+    identifiers = [ "id", "region" ]
+    iac {
+      terraform {
+        type = "aws_imagebuilder_image"
+        identifiers = [
+          "output_resources.#.amis.#.image|@flatten|0",
+          "output_resources.#.amis.#.region|@flatten|0",
+        ]
+      }
+    }
+  }
+
+  resource "ec2.instances" {
+    ignore_attributes = ["launch_time"]
+
+    iac {
+      terraform {
+        type = "aws_instance"
+      }
+    }
+  }
+
+  resource "ec2.internet_gateways" {
+    filters = [
+      "NOT EXISTS (SELECT 1 FROM aws_ec2_internet_gateway_attachments a JOIN aws_ec2_vpcs v ON v.id=a.vpc_id WHERE a.internet_gateway_cq_id=c.cq_id AND v.is_default)",
+      "NOT EXISTS (SELECT 1 FROM aws_ec2_route_table_routes WHERE gateway_id=c.id AND destination_cidr_block='0.0.0.0/0')"
+    ]
+
+    iac {
+      terraform {
+        type = "aws_internet_gateway"
+      }
+    }
+  }
+
+  resource "ec2.nat_gateways" {
+    ignore_attributes = [ "arn", "create_time", "state", "vpc_id" ]
+    iac {
+      terraform {
+        type = "aws_nat_gateway"
+      }
+    }
+  }
+
+  resource "aws_ec2_nat_gateway_addresses" {
+    identifiers = [ "parent.id", "c.network_interface_id" ]
+    iac {
+      terraform {
+        type = "aws_nat_gateway"
+        identifiers = [ "id", "network_interface_id" ]
+      }
+    }
+  }
+
+  resource "ec2.network_acls" {
+    filters = [ "c.is_default!=true" ]
+
+    iac {
+      terraform {
+        type = "aws_network_acl"
+      }
+    }
+  }
+
+  #        resource "aws_ec2_network_acl_entries" {
+  #            # TODO: no CRC32 function, no data in tests to verify
+  #            identifiers = [ sql("CONCAT('nacl-',CRC32(CONCAT(parent.id,'-',c.rule_number,'-',CASE WHEN c.egress THEN 'true' ELSE 'false' END,'-',c.protocol,'-')))") ]
+  #            filters = [ "((c.cidr_block='0.0.0.0/0' AND c.rule_number=32767) OR (c.ipv6_cidr_block=':/0' AND c.rule_number=32768)) AND c.rule_action='deny' AND c.protocol='-1'" ]
+  #
+  #            iac {
+  #                terraform {
+  #                    type = "aws_network_acl_rule"
+  #                }
+  #            }
+  #        }
+
+  resource "ec2.regional_config#key" { # TODO: add account/region support
+    identifiers = [ "ebs_default_kms_key_id" ]
+    ignore_attributes = [ "ebs_encryption_enabled_by_default" ]
+    iac {
+      terraform {
+        type = "aws_ebs_default_kms_key"
+        identifiers = [ "key_arn" ]
+      }
+    }
+  }
+
+  resource "ec2.regional_config#ebs" { # TODO: add account/region support
```

Comment:
open issue for these ?

Reply:
https://github.com/cloudquery/cloudquery/issues/566